package/MDAnalysis/lib/nsgrid.pyx

```diff
@@ -88,21 +89,22 @@ DEF EPSILON = 1e-5
 
 ctypedef np.int_t ns_int
 ctypedef np.float32_t real
+ctypedef np.float64_t dreal
```

Comment:
Is there a performance different for moving to double precision?  (probably yes?) If so, maybe we should open an issue about precision in our distance calculations.  Especially as for this Cython it's easy to used a fused type (aka template)

Reply:
Good call, I'll run some benchmarks.